spec/std/enumerable_spec.cr

```diff
@@ -1219,12 +1219,46 @@ describe "Enumerable" do
     it "returns a hash with counts according to the value returned by the block" do
       %w[a A b B c C C].tally_by(&.downcase).should eq({"a" => 2, "b" => 2, "c" => 3})
     end
+
+    context "with hash" do
+      it "returns a hash with counts according to the value returned by the block" do
```

Comment:
Now that I think about it, there's no test where the hash has some initial values. Could you add one for completeness?

Reply:
@asterite yes, I will. `tally` method ignores the default value like in Ruby implementation https://github.com/ruby/ruby/pull/4318/files#diff-21a883d5607ae36958447ba2e0436f8b5cce1ba244d60fac9628b43aea8fc58dR48

Reply:
I think the comment was about the hash already being filled with some actual values. At least that's what I'd be interested to see.
The spec for default value is also good, though 👍

Reply:
added